pkg/azuredisk/azure_common_linux.go

```diff
@@ -28,17 +28,20 @@ import (
 	"k8s.io/kubernetes/pkg/util/mount"
 )
 
-// exclude those used by azure as resource and OS root in /dev/disk/azure
+// exclude those used by azure as resource and OS root in /dev/disk/azure, /dev/disk/azure/scsi0
+// "/dev/disk/azure/scsi0" dir is populated in Standard_DC4s/DC2s on Ubuntu 18.04
```

Comment:
What about other distros, e.g. Ubuntu 16 or REHL? Are they always be `/de/disk/azure`?

Reply:
if waagent is installed on those distros, it's always `/dev/disk/azure`

Reply:
cool